### DISCUSSION TITLE
Your thoughts on multilingual version of GO terms

### DISCUSSION THREAD

**Discussion by linikujp** [geneontology/helpdesk#341] [Ideas]
**Please describe your question, idea, or concern.**
Dear GO developers,
As I am involved in an ontology (COVID19 related) translation project, and this ontology uses a lot of GO terms. Within the team, we have different opinion on translating GO terms into other languages than English. I am wondering, have you thought about this, and what is your position and recommendations on this?
Your input will be greatly appreciated.
Thank you!

**Comment by cmungall** [comment]
I am not aware of any projects to internationalize GO.

The Human Phenotype Ontology has translations into multiple languages. @drseb led these efforts. See 

 - https://crowdin.com/project/hpo-translation
 - https://hpo.jax.org/app/help/mapping

There may be techniques you can use here.

Wikidata has translations of many concepts to different languages (and is in fact language neutral). They have mappings to GO (not sure of completeness). I wrote a blog piece on how this can be leveraged for ontologies: https://douroucouli.wordpress.com/2020/02/26/using-wikidata-for-crowdsourced-language-translations-of-ontologies/

Many GO concepts are composition and are defined by OWL axioms that follow defined patterns using the DOSDP system. This could be leveraged for translation.

Unfortunately we cannot dedicate any resources to help with this, but we would be interested in what you learn (is there a ticket on th CIDO tracker we can make a reciprocal link to?). Feel free to ask any further questions though.

**Comment by linikujp** [comment]
Great! The TXPO team has translated some GO terms into Japanese, and there are efforts undergoing translate some terms into other languages(Japanese, Chinese, Arabic and other European languages). I am wondering if GO would like to take the translations that have been done for other projects. Or, should we put some translations into wikidata? @cmungall

**Comment by akodamullil** [comment]
Hello Asiyah and Chris,
As a part of making multilingual Covid 19 Ontology (http://bioportal.bioontology.org/ontologies/COVID-19), we will converting many GO terms to more than 10 languages. We will be happy to collaborate and share the resources. 
Alpha
